spec/policies/idv/gpo_verify_by_mail_policy_spec.rb

```diff
@@ -106,6 +106,46 @@
           expect(subject.send_letter_available?).to eq(true)
         end
       end
+
+      context 'user has a pending in-person enrollment' do
+        let!(:in_person_enrollment) { create(:in_person_enrollment, :pending, user: user) }
+
+        it 'returns false when the feature flag is enabled' do
+          allow(IdentityConfig.store).to receive(
+            :no_verify_by_mail_for_biometric_comparison_enabled,
+          ).and_return(true)
+
+          expect(subject.send_letter_available?).to eq(false)
+        end
+
+        it 'returns true when the feature flag is disabled' do
+          allow(IdentityConfig.store).to receive(
+            :no_verify_by_mail_for_biometric_comparison_enabled,
+          ).and_return(false)
+
+          expect(subject.send_letter_available?).to eq(true)
+        end
+      end
+
+      context 'user has an establishing in-person enrollment' do
+        let!(:in_person_enrollment) { create(:in_person_enrollment, :establishing, user: user) }
+
+        it 'returns false when the feature flag is enabled' do
+          allow(IdentityConfig.store).to receive(
+            :no_verify_by_mail_for_biometric_comparison_enabled,
+          ).and_return(true)
+
+          expect(subject.send_letter_available?).to eq(false)
+        end
+
+        it 'returns true when the feature flag is disabled' do
+          allow(IdentityConfig.store).to receive(
+            :no_verify_by_mail_for_biometric_comparison_enabled,
+          ).and_return(false)
+
+          expect(subject.send_letter_available?).to eq(true)
+        end
+      end
```

Comment:
I ran through IPP flow coming in from Sinatra (picking Biometric).  When the flag was true, GPO was not offered on `/verify/phone`. When the flag was false, GPO was offered. I also continued on to `/verify/phone_confirmation` and clicked on "Use another phone number" to confirm expected behavior. The spec set up and expected results align with manual testing scenarios.  LGTM